doc/developer-notes.md

```diff
@@ -338,9 +338,10 @@ Recommendations:
 
 ### Generating Documentation
 
-The documentation can be generated with `cmake --build <build_dir> --target docs`.
-The resulting files are located in `<build_dir>/doc/doxygen/html`; open
-`index.html` in that directory to view the homepage.
+Assuming the build directory is named `build`,
+the documentation can be generated with `cmake --build build --target docs`.
```

Comment:
Not sure that making this less generic is an improvement?

Reply:
Seems to me that was the way we've preceded before: https://github.com/bitcoin/bitcoin/blob/6ce50fd9d0ae6850d54bf883e7a7c1bcb6912c5c/src/test/README.md?plain=1#L19

And since we need to update the build directories in the commits (at least that's what I tried doing here), we need a sensible default.

Do you have a better suggestion?

Reply:
> thus no update is required

But isn't that how we've done it in other cases, i.e. assume a default build dir?
Or you mean that the changes here from `/src/` to `/build/src/` are all invalid?